models/models.js

```diff
@@ -12,72 +13,109 @@ exports.selectEndpoints = () => {
 
 exports.selectArticleById = (article_id) => {
     return db.query('SELECT * FROM articles WHERE article_id = $1', [article_id])
-    .then((results) => {
-        if (results.rows.length === 0){
-           return Promise.reject({
-                status: 404,
-                msg: 'article does not exist'
-            })
-        }
-        return results.rows[0];
-    })
+        .then((results) => {
+            if (results.rows.length === 0) {
+                return Promise.reject({
+                    status: 404,
+                    msg: 'article does not exist'
+                })
+            }
+            return results.rows[0];
+        })
 }
 
-exports.selectArticles = () => {
-    return db.query('SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.article_id) AS comment_count FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id GROUP BY articles.article_id ORDER BY created_at DESC')
-    .then((results) => {
-        return results.rows
-    })
+exports.selectArticles = (sort_by, order) => {
+    let queryStr = "SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.article_id) AS comment_count FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id GROUP BY articles.article_id"
+    const validColumns = [
+        "title",
+        "topic",
+        "author",
+        "body",
+        "created_at",
+        "article_img_url"
+    ]
+    const validOrders = ["asc", "desc"]
```

Comment:
Are you intentionally being case insensitive here? Or would you want to allow someone to make a request to order by ascending using "ASC" as the value to the query?